observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,202 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-forwarding-logs"]
+= Forwarding Logs to a LokiStack instance
+include::_attributes/common-attributes.adoc[]
+:context: otel-forwarding-logs
+
+You can deploy and configure the OpenTelemetry Collector to forward logs to a LokiStack instance. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters.
+
+This use of the Loki Exporter is a temporary Technology Preview feature, which is planned to be deprecated with the publication of an improved solution in which the Loki Exporter is replaced with the OTLP HTTP Exporter.
+
+:FeatureName: The Loki Exporter
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* The {OTELOperator} is installed.
+* The {loki-op} is installed. For more information, see xref:../logging/log_storage/installing-log-storage.adoc#installing-log-storage[Installing LokiStack log storage].
```

Comment:
```suggestion
* The {loki-op} is installed. 
```
For more information, see xref:../logging/log_storage/installing-log-storage.adoc#installing-log-storage[Installing LokiStack log storage].

Can just put this in additional resources after the module.

Reply:
Yes, I considered putting it there. Considering that this feature is closer to a Developer Preview, it can be handy in the prerequisites.